wolfcrypt/src/rsa.c

```diff
@@ -3952,30 +3952,49 @@ int wc_CheckProbablePrime(const byte* pRaw, word32 pRawSz,
 int wc_MakeRsaKey(RsaKey* key, int size, long e, WC_RNG* rng)
 {
 #ifndef WC_NO_RNG
-    mp_int p, q, tmp1, tmp2, tmp3;
+#ifdef WOLFSSL_SMALL_STACK
+    mp_int *p = (mp_int *)XMALLOC(sizeof *p, key->heap, DYNAMIC_TYPE_RSA);
```

Comment:
Where are these mallocs checked for success?

Reply:
Oh dear, the check was missing.  I just reviewed the wolfcrypt/ diff for any other missed checks and couldn't find any.  Thanks much!